blocks.js

```diff
@@ -7553,13 +7554,41 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-            // } else if (key.indexOf('§_def') === 0) {
```

Comment:
> (I guess we can actually remove these two lines, since they've been commented out for ages anyway)

oh, that would make me very happy. 

But BH says his indents look fine to him. It looks like he's using actual tab chars...